pkg/common/copy.go

```diff
@@ -39,8 +46,9 @@ func CopyFolder(src, dst string, config *CopyConfig) (CopyStats, error) {
 			return nil
 		}
 
+		writePath := dstDir + relPath
```

Comment:
it's always better form when working with paths to use the path manipulation methods from standard library to avoid issues with leading / trailing slashes.
```suggestion
		writePath := filepath.Abs(filepath.Join(dstDir, relPath))
```